SkipList/SkipList/SkipList.cs

```diff
@@ -0,0 +1,319 @@
+﻿namespace SkipList;
+
+using System.Collections;
+
+/// <summary>
+/// SkipList realization of IList interface
+/// </summary> 
+public class SkipList<T> : IList<T> where T : IComparable <T>
+{
+    private SkipNode head;
+    private int level { get; set; }
+    private int maxLevel;
+    public int Count { get; private set; } 
+    public bool IsReadOnly { get; }
+    private readonly double probability;
+    private static readonly Random random = new();
+
+    /// <summary>
+    /// SkipList constructor
+    /// <param name="maxLevel">Maximum level of list</param>
+    /// </summary> 
+    public SkipList(int maxLevel) {
```

Comment:
В C# фигурная скобка в методах должна быть одна на строке:
```suggestion
    public SkipList(int maxLevel)
    {
```